src/main/resources/templates/slurm/command/scontrolCommand

```diff
@@ -0,0 +1,10 @@
+scontrol [(${command})]
+[# th:if="${partitionName != null}"]
```

Comment:
let's change to `${not #strings.isEmpty(partitionName)}`

Reply:
fixed